devtools/gradle/pom.xml

```diff
@@ -21,30 +21,7 @@
         <skip.gradle.build>false</skip.gradle.build>
```

Comment:
When do we envision that this will be set? Is it even needed anymore?

Reply:
This is useful for speeding up the build when you don't need to test the Gradle plugin. I use it sometimes but I am not opposed to removing it if necessary

Reply:
Just asking to understand if it still makes sense, I'm not saying we should remove it